client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Button from 'components/button';
+import { translate } from 'i18n-calypso';
+import { setJPOSummary } from 'state/signup/steps/jpo-summary/actions';
+
+const JPOSummaryStep = React.createClass( {
+	propTypes: {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setJPOSummary: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	},
+
+	getFormattedPayload() {
+		let payload = this.props.signupProgress;
+		
+		let formattedPayload = {
+			siteTitle: payload[0].jpoSiteTitle.siteTitle,
+			siteDescription: payload[0].jpoSiteTitle.siteDescription,
+			businessPersonal: payload[1].jpoSiteType.businessPersonal,
+			genre: payload[1].jpoSiteType.genre,
+			businessName: payload[1].jpoSiteType.addressInfo.businessName,
+			businessAddress: payload[1].jpoSiteType.addressInfo.streetAddress,
+			businessCity: payload[1].jpoSiteType.addressInfo.city,
+			businessState: payload[1].jpoSiteType.addressInfo.state,
+			businessZipCode: payload[1].jpoSiteType.addressInfo.zipCode,
+			homepageFormat: payload[2].jpoHomepage,
+			addContactForm: payload[3].jpoContactForm
+		};
+
+		return formattedPayload;
+	},
+
+	completeOnboarding() {
+		console.log( this.getFormattedPayload() );
+	},
+
+	goToStepOne() {
+		this.props.goToStep( 1 );
+	},
+
+	goToStepTwo() {
+		this.props.goToStep( 2 );
+	},
+
+	goToStepThree() {
+		this.props.goToStep( 3 );
+	},
+
+	goToStepFour() {
+		this.props.goToStep( 4 );
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 } icon="checkmark" />;
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header">Steps you've completed:</div>
+								<ul>
+									{ 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle ? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[1].jpoSiteType ? <li>{ checkMark } { translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[2].jpoHomepage ? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[3].jpoContactForm ? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li> : null }
+									{ connectionToJetpackComplete ? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li> : null }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Jetpack Connection' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).